src/pudl/metadata/classes.py

```diff
@@ -735,7 +735,7 @@ def to_sql(  # noqa: C901
         return sa.Column(
             self.name,
             self.to_sql_dtype(),
-            *[sa.CheckConstraint(check) for check in checks],
+            *[sa.CheckConstraint(check, hash(check)) for check in checks],
```

Comment:
What's going on here? I can't tell what role `hash(check)` is supposed to be playing based on the SQLAlchemy docs.